apis/filters/parse.go

```diff
@@ -150,3 +157,33 @@ func FromParam(p string) (Args, error) {
 	}
 	return args, nil
 }
+
+//FromFilterOpts parse key=value to Args string from cli opts
```

Comment:
Maybe we can add some test cases for this new function, WDYT ? @ZYecho

Reply:
good idea, I add a testcase to cover this.